src/components/Chips/Chips.tsx

```diff
@@ -104,6 +120,7 @@ const Chips: VibeComponent<ChipsProps, HTMLElement> & {
         id={id}
         style={backgroundColorStyle}
         onMouseDown={onMouseDown}
+        onClick={onClick}
```

Comment:
we know how to this accessible so let's not do it in a different pr (it's part of the definition of done)
You can add a clickable conditionally (just be aware that there is a different name to the data test id attribute0